src/functions.js

```diff
@@ -89,11 +91,14 @@ itemsDisplay.addEventListener('click', (e) => {
 
 // clear all tasks
 const eraseAll = () => {
-  todos.length = 0;
+  todos = todos.filter((todo) => !todo.completed);
```

Comment:
- [ ] I noticed that when I checked a few todo items as true and tried to delete the selected items, all todos get cleared at once irrespective of the todos I selected. Kindly fix it 👍🏽